traefik/values.yaml

```diff
@@ -159,6 +159,22 @@ ingressRoute:
     middlewares: []
     # -- TLS options (e.g. secret containing certificate)
     tls: {}
+  healthcheck:
+    # -- Create an IngressRoute for the healthcheck probe
+    enabled: true
```

Comment:
```suggestion
    enabled: false
```

Reply:
Alright that was my bad i didn't notice it